src/index.ts

```diff
@@ -19,7 +19,6 @@ export type {PositionProps, AbsoluteProps, FixedProps, RelativeProps, StickyProp
 
 // Hooks
 export {default as useDetails} from './hooks/useDetails'
-export {default as useMouseIntent} from './hooks/useMouseIntent'
```

Comment:
This is technically a breaking change since it's no longer exported.  How do we handle that on this project?

Reply:
When you create a changeset you can label this PR as a `major` change 👍